uint32 word
float64 value
